tox.ini

```diff
@@ -24,28 +22,23 @@ passenv = *
 description = Run tests and coverage report
 extras =
     test
-commands = 
+commands =
     coverage erase
     coverage run -m pytest -vv {posargs: pydoctor}
     coverage report -m
+    # The XML version is generatred to be uploaded to Codecov.
+    coverage xml -o coverage.xml -i
```

Comment:
Do you use `tox -e test` during normal development ?

If yes we night want to only run the coverage for CI environment

Reply:
Are you ok with how fast the test are executed via tox ?

Do you always want to get the coverate report during local development?

I am now sure if `coverage report -m` is usefull for each local test run.

For normal TDD development, I alwasy create a separate virtualenvironmant and run the tests outside of tox... it's so much faster.

Reply:
Now with xml file generatation it is even slower. 

Maybe we can have a `tox -e test` and another `tox -e test-coverage` ?

If you are ok with the current tox env, I am fine :)

I am not using tox, outside of the CI build :p